scripts/update.sh

```diff
@@ -2,6 +2,9 @@
 # vim: noexpandtab
 
 main() {
+	# add some jitter to prevent overloading the packaging machines
+	sleep $(( RANDOM % 900 ))
```

Comment:
put it after the export DEBIAN_FRONTEND statement

Reply:
it's not necessary since the only thing that uses that env var is apt-get